nginx-hello-nonroot/plain-text-version/Makefile

```diff
@@ -0,0 +1,15 @@
+VERSION=0.1
+PREFIX=nginxdemos/nginx-hello
+TAG=$(VERSION)
+
+all: build push test
+
+build:
```

Comment:
the makefile wasn't tested at all - it fails :(

Reply:
if it failed it's because of `PREFIX=nginxdemos/nginx-hello` no credentials to push to registry or because an existing container with name `nginx-hello` is already running. I tested it on my side using my docker registry it works fine. What should we do about the PREFIX? change it to just `nginx-hello` ?